src/client.js

```diff
@@ -151,9 +150,9 @@ const update = () => {
             event.preventDefault();
             return false;
         };
-        canvas.addEventListener("touchstart", onMouseDown);
-        canvas.addEventListener("touchend", onMouseUp);
-        canvas.addEventListener("touchmove", onMouseMove);
+        canvas.addEventListener("mousedown", onMouseDown);
+        canvas.addEventListener("mouseup", onMouseUp);
```

Comment:
Both `mouseup` and `mousemove` should be on the document, not just the canvas. In particular `mouseup` won't be called if they press the canvas, drag off of it and then release.